perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -111,6 +111,19 @@
 	 * @throw ResourceNotExistsException When Resource with <code>id</code> doesn't exist.
 	 */
 	/*#
+	 * Returns all specified Group-Resource attributes for selected Group and Resource.
+	 * If <code>attrNames</code> is empty, it returns all non-empty attributes.
+	 * If <code>workWithGroupAttributes == true</code> then also Group attributes are returned. <code>False</code> is default.
```

Comment:
*This is not true, since this method takes list of attrNames => it returns also empty attributes for both group_resource and group namespace if workWithGroupAttributes == true. If passed list of names is empty, it should return all non-empty attributes. Please also fix spaces in above line. Thank you.*

The javadoc was updated. Again it seems I deleted your previous comment. 
I will not use --amend in the future, but rather squash commits at the end of review.